spot_rl_experiments/spot_rl/envs/nav_env.py

```diff
@@ -25,21 +25,63 @@
 
 def parse_arguments(args=sys.argv[1:]):
     parser = get_default_parser()
-    parser.add_argument("-g", "--goal")
-    parser.add_argument("-w", "--waypoints")
-    parser.add_argument("-d", "--dock", action="store_true")
-    parser.add_argument("-rt", "--record_trajectories", action="store_true")
-    parser.add_argument("-st", "--save_trajectories")
+    parser.add_argument(
+        "-g", "--goal", help="input:string -> goal x,y,theta in meters and radians"
+    )
+    parser.add_argument(
+        "-w",
+        "--waypoints",
+        help="input:string -> nav target waypoints (comma seperated) where robot needs to navigate to",
+    )
+    parser.add_argument(
+        "-d",
+        "--dock",
+        action="store_true",
+        help="input: not needed -> should dock the robot after navigation",
```

Comment:
```suggestion
        help="input:boolean -> True/False",
```

Reply:
We don't need any input for this from the user.
We are storing them with "store_true" action.

Reply:
addressed